doc/pages/blurbs.json

```diff
@@ -682,5 +682,14 @@
     "link_name": "manishsharma.ml",
     "link": "https://manishsharma.ml",
     "social": "globe"
+  },
+  {
+    "name": "Nikhil Bhatt",
+    "image": "/assets/contributors/nikhil_bhatt.jpg",
+    "profile": "Techie, who loves to solve the real world problems using code. Currently working as a Full stack developer in Ruby on Rails and React",
```

Comment:
Grammar fix: `Currently, I'm working as a full-stack developer in Ruby on Rails and React.`